modules/nf-core/bbmap/filterbyname/tests/nextflow.config

```diff
@@ -0,0 +1,24 @@
+process {
+    withName: 'test_bbmap_filterbyname_single_end:BBMAP_FILTERBYNAME' {
+        ext.args = [
```

Comment:
How is this working? `ext.args` should be a `String`, rather than a `Map`

Reply:
This wasn't being used any more